# Estimated states of WBR

float64 pitch
float64 pitch_rate
float64 vel
float64 yaw_rate